src/test/java/basic/Exercise05Test.java

```diff
@@ -0,0 +1,19 @@
+package basic;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class Exercise05Test {
+    @Test
+    void shouldCalculateFactorial_OK() {
```

Comment:
The test name should reflect the method name
```suggestion
    void shouldFactorial_OK() {
```